doc/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 0.5.7
+-------------
+
+* Change ``warn`` to ``warning`` in `bm_logging.py`
```

Comment:
Please mention that in Python 2, Logger.warn() calls warnings.warn() and so is slower than Logger.warning().

By the way, can you please please post results of Logger.warn() and Logger.warning()?